scripts/postinstall

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Could we call this script `copy-katex-assets` instead of postinstall? We'll have more later on, would rather chain them with `&&` than stick everything into one.